lib/attrsets.nix

```diff
@@ -1167,7 +1167,9 @@ rec {
   recurseIntoAttrs =
     # An attribute set to scan for derivations.
     attrs:
-    attrs // { recurseForDerivations = true; };
+    if (attrs != null)
+    then attrs // { recurseForDerivations = true; }
+    else null;
```

Comment:
I don't think changing `[dont]recurseIntoAttrs` for this is justifiable..

Reply:
Dropped `lib/` changes and switched to `llvmPackages = {};`. That fixes eval for me as well.